### DISCUSSION TITLE
can useIsFetching return loading status of non-get reqeusts

### DISCUSSION THREAD

**Discussion by mwmcode** [TanStack/query#269] [General]
Hi Tanner,

Thanks for the awesome lib.

Question: can we get [isFetching](https://github.com/tannerlinsley/react-query#displaying-global-background-fetching-loading-state) to consider requests other than `GET`?

**Comment by cherniavskii** [comment]
You can create your own `useMutation` hook and separate `useIsMutating` hook.
This way you'll be able to track fetches and mutations globally.

**Reply by mwmcode** [reply]
Good idea. 
Currently the whole global indicator is causing React `16.13.1` to throw warnings

_Cannot update a component while rendering a different component_

Will give it a try, thanks.
